data/mods/ssb/conditions.ts

```diff
@@ -245,6 +245,24 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			this.add(`c:|${getName('Scotteh')}|\`\`Segmentation fault (core dumped)\`\``);
 		},
 	},
+	sharpclaw: {
+		noCopy: true,
+		onStart(pokemon) {
+			if (pokemon.species.name === 'Sneasel') {
+				this.add(`c:|${getName('sharp_claw')}|Hi, I'm Tumble! hf :D`);
+			} else {
+				this.add(`c:|${getName('sharp_claw')}|Hi, I'm Rough! gl >:)`);
+			}
+		},
+		// onSwitchOut implemented in ability instead
```

Comment:
ik its being pedantic but please keep all of the big quotes in one place (e.g. what if irpa removes sharp_claw's ability?)

Reply:
I only did this so the quote would consistently appear before the set change message. I guess I could use priorities?

Reply:
You can just give the ability a negative priority, I think that will be fine

Reply:
Worst case scenario, we can either keep it as is or ask sharp if they want the forme change effect as an innate

Reply:
priority should work. another thing to note though is that it is currently unavoidable for the set change to be announced which is probably how one would expect the ability to work (like palafin). might be a low priority change for later

Reply:
yeah, sounds good